content/developers/_index.md

```diff
@@ -4,8 +4,34 @@ kind: documentation
 customnav: developersnav
 ---
 
-* [Consult our API](/api)
-* [Learn about DogstatsD](/developers/dogstatsd)
-* [Consult all available librairies](/developers/libraries)
-* [Learn more about metrics behaviour](/developers/metrics)
-* [Learn more about metrics types](/developers/metrictypes)
\ No newline at end of file
+Monitoring tools should be flexible, so we provide multiple ways to interact with Datadog. This section will give you a quick overview of the various methods available
+
+## API
+Datadog features a powerful API to programmatically access the service. You can submit custom metrics; manage dashboards and monitors; schedule downtimes; and perform administrative actions.
+Available endpoints can be found on [our dedicated API documentation page](/api).
+
+In addition, [many libraries](/developers/libraries/#api-and-dogstatsd-client-libraries) are available to directly interact with the API.
+
+## Submitting custom metrics
+While you can submit metrics directly through [our API](/api), you can also submit metrics via the Datadog Agent using DogStatsD and custom checks.
+
+* The Datadog Agent includes DogStatsD, a powerful statsd daemon with additional features, that provides better control over your metric metadata and aggregation.
```

Comment:
Let's remove the bullet points on this and the next item. They read more like standard paragraphs now.